src/course-home/outline-tab/Section.jsx

```diff
@@ -2,11 +2,11 @@ import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { Collapsible, IconButton, Icon } from '@openedx/paragon';
-import { faCheckCircle as fasCheckCircle, faMinus, faPlus } from '@fortawesome/free-solid-svg-icons';
+import { faCheckCircle as fasCheckCircle } from '@fortawesome/free-solid-svg-icons';
```

Comment:
[question] Should we replace `faCheckCircle` icon with paragon icon as well?

Reply:
> [question] Should we replace `faCheckCircle` icon with paragon icon as well?

There is no issues with them, since these icons are used as <FontAwesomeIcon>, and not in the <IconButton> component as props, where they are deprecated. So decided not to touch them or change.